ecosystem/typescript/sdk/src/plugins/ans_client.ts

```diff
@@ -132,6 +133,93 @@ export class AnsClient {
     return pendingTransaction.hash;
   }
 
+  /**
+   * Mint a new Aptos Subdomain
+   *
+   * @param account AptosAccount the owner of the domain name
+   * @param subdomainName subdomain name to mint
+   * @param domainName Aptos domain name to mint under
+   * @param expirationTimestamp must be set between the domains expiration and the current time
+   * @returns The hash of the pending transaction submitted to the API
+   */
+  async mintAptosSubdomain(
+    account: AptosAccount,
+    subdomainName: string,
+    domainName: string,
+    expirationTimestamp: number,
```

Comment:
Could we go for a more explicit name, like `expirationUnixtimeSecs` or something?

Reply:
sure